registry/etcdv3/registry.go

```diff
@@ -0,0 +1,324 @@
+package etcdv3
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"path"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+)
+
+import (
+	"github.com/apache/dubbo-go/common"
+	"github.com/apache/dubbo-go/common/constant"
+	"github.com/apache/dubbo-go/common/extension"
+	"github.com/apache/dubbo-go/common/logger"
+	"github.com/apache/dubbo-go/common/utils"
+	"github.com/apache/dubbo-go/registry"
+	"github.com/apache/dubbo-go/remoting/etcdv3"
+	"github.com/apache/dubbo-go/version"
+)
+
+import (
+	"github.com/juju/errors"
```

Comment:
We agreed to use only `perrors`.  Is it `github.com/juju/errors`  necessary ？

Reply:
@fangyincheng agree with u.

Reply:
Thx, i had change github.comjuju/errors to  github.com/pkg/errors